### DISCUSSION TITLE
Alateen "Type" on backend appearing as "Young Person" on frontend

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#1241] [Q&A]
Do you know why when tagging a type on the WordPress backend i.e. "Alateen", it appears as "young Person" on the website.

View link: 
https://staging.al-anon.org.au/find-a-meeting/?type=young-people&search=alateen

Our users will be looking for "Alateen" meetings on this page and searching for alateen so we want to make this clear by using the same terminology.

Please confirm if this is possible.

Cheers,
Natalie

**Comment by tim-rohrer** [comment]
In looking at the code, assuming your site's program is set to Al-Anon, there is no type label of `young-people`. There is `young-adults` with a type abbreviation of `ya`, while the abbreviation of `Y` should yield  Alateen meetings. 

It is unclear as to why your link includes `type=young-people`? I suspect there is a conflict of types, or perhaps the code is picking the closest match (i.e., starts with 'y'). 

I tried changing the link to `?type=y`, but no meetings are shown then, and I'm not sure what you actually have in your database.

Do you have Al-Anon selected as the program type?
Do you have any custom/overrides programmed for types?

**Reply by tim-rohrer** [reply]
Hey @nataliefaith2, Another C4R member thinks the issue may be related to what is likely custom code to make `find-a-meeting` work as a URL. 

Since this is a staging site, can you experiment with the short code (occurs when you select TSML-UI from the settings page) or the standard URL at `/meetings`? You'll need to select a Wordpress time zone (not UTC offset) for this to work.

**Reply by nataliefaith2** [reply]
Hi Tim,

Thanks for your message and responding so quickly.

I've used a shortcode and set up the /find-a-meeting because the standard URL /meetings has a timezone issue that isn't working:
[https://staging.al-anon.org.au/meetings/](https://staging.al-anon.org.au/meetings/) 

Yes, Al-Anon is selected. See screenshot below. 
<img width="1134" alt="Screen Shot 2023-10-27 at 10 15 45 pm" src="https://github.com/code4recovery/12-step-meeting-list/assets/23748349/a9c702ba-9374-45b0-8c29-74678bf02908">

I hear what you're saying but I selected Alateen in "New Meeting" and it appears on the live page as Young people.
https://staging.al-anon.org.au/find-a-meeting/?type=young-people
<img width="1128" alt="Screen Shot 2023-10-27 at 10 20 47 pm" src="https://github.com/code4recovery/12-step-meeting-list/assets/23748349/13b0323e-b741-421d-a126-11be2600398b">

**I haven't used any overrides. This is the only code I used is this (and only on the Find A Meeting page):**

< div id="tsml-ui"
  data-src="/wp-admin/admin-ajax.php?action=meetings"
  data-mapbox=
"pk.eyJ1IjoibmF0YWxpZWZhaXRoMiIsImEiOiJjbGUzZzR2YjIwOGtuM3dvNWt6aW5jMnlyIn0.x7y7LZcPNuC7mgQpGYrc9A"
  data-timezone= "Australia/Melbourne" >
</ div>

<script src="https://tsml-ui.code4recovery.org/app.js" async></script>
<script>

var tsml_react_config = {
  distance_unit: 'km',
};
</script>

Is there anything I can add to this code (or takeaway) that could amend this issue?

**Comment by nataliefaith2** [comment]
> Hey @nataliefaith2, Another C4R member thinks the issue may be related to what is likely custom code to make `find-a-meeting` work as a URL.
> 
> Since this is a staging site, can you experiment with the short code (occurs when you select TSML-UI from the settings page) or the standard URL at `/meetings`? You'll need to select a Wordpress time zone (not UTC offset) for this to work.

Thank you for your feedback. Yes, I have the timezone selected within WP but it's just not working. This is why I began using the custom code. [https://staging.al-anon.org.au/meetings/](https://staging.al-anon.org.au/meetings/)

I can only assume that the divi theme must be disrupting the usual functionality of this plugin? - Do you know of any plugins or themes that cause bugs with the plugin? 

Cheers,

**Reply by joshreisner** [reply]
<img width="794" alt="Screenshot 2023-10-27 at 7 32 41 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/29f7cc30-f5d1-428f-92fa-7e6d95d7c425">

hi @nataliefaith2 - do you have `Melbourne` selected at WordPress Dashboard > Settings?

**Reply by nataliefaith2** [reply]
I had it in UTC, but have now selected Melbourne and will test. 
<img width="934" alt="Screen Shot 2023-10-28 at 11 45 44 am" src="https://github.com/code4recovery/12-step-meeting-list/assets/23748349/266b7981-3327-49d4-b0a7-b668e46f9990">

**Reply by joshreisner** [reply]
@nataliefaith2 great so now that the meetings page is working, we suggest archiving the `/find-a-meeting` page

the issue with near me should be answered over here: https://github.com/code4recovery/12-step-meeting-list/discussions/1231#discussioncomment-7242078 we recommend updating the target of the "Find Your Closest Meeting" to `/meetings/?distance=10&mode=me` - this links to the "Near Me" mode activated

**Reply by nataliefaith2** [reply]
Thank you, I've updated my menu to this link. 
And thank you for the comment resource.

**Comment by nataliefaith2** [comment]
Okay that worked, I feel very stupid now haha - thank you!

**Comment by nataliefaith2** [comment]
Will this also help with the "Near me" issue we've been having where it defaults to Melbourne, when the person isn't in melbourne and there's a meeting much closer to them?
